lib/airbrake-ruby/notice.rb

```diff
@@ -62,7 +62,7 @@ class Notice
     # @since v1.7.0
     # @return [Hash{Symbol=>Object}] the hash with arbitrary objects to be used
     #   in filters
-    attr_reader :stash
+    attr_reader :stash, :exception
```

Comment:
We should probably move this on a separate line because the comment above describes `:stash`.